server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -93,6 +93,7 @@
 
     private static final String MDC_REQUEST_TYPE_KEY = "requestType";
     private static final String MDC_REQUEST_UUID_KEY = "requestUuid";
+    private static final String MDC_REQUEST_JOB_KEY = "jobKey";
```

Comment:
```MDC_JOB_KEY_KEY```

The name sounds a bit silly and redundant, but it is the key for the job key.

Reply:
Done.